soc/intel/intel_adsp/ace/Kconfig.soc

```diff
@@ -32,9 +32,9 @@ config SOC_SERIES
 config SOC_TOOLCHAIN_NAME
 	default "intel_ace15_mtpm" if SOC_INTEL_ACE15_MTPM
 	default "intel_ace15_mtpm" if SOC_INTEL_ACE20_LNL
-	default "intel_ace30_ptl" if SOC_INTEL_ACE30_PTL
+	default "intel_ace30" if SOC_INTEL_ACE30
```

Comment:
Also, keep the toolchain name as `intel_ace30_ptl` as it is related to the Zephyr SDK, and the toolchain there is named with `_ptl`. Changing this would mean Zephyr SDK can no longer build this platform.